### ISSUE TITLE
Accessibility Testing for CTO Health Team: MyHealtheVet on VA.gov: Landing Page

### ISSUE THREAD

**Issue by MarciMcGuire** [department-of-veterans-affairs/va.gov-team#54863]
### Product artifacts

- [X] Staging URL https://staging.va.gov/my-health/
- [X] Specify which pages are included in the review       
   - After logging into VA.gov on Staging using the below-referenced credentials, click on the "My Health" link in the main nav to access the landing page.  This is the only page in scope for the staging review.
- [X] Staging test user information  - test user information can be obtained through the [Test User Dashboard](https://tud.vfs.va.gov/) and selecting a user that has an MHV account.

### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

Marci McGuire

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Testing went well; we are using Design System guidelines and components.

### Who completed the axe scans?

Marci McGuire

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results
https://axe.deque.com/issues/a05269d9-fbec-4d12-a754-fab4d10583db
https://axe.deque.com/issues/47167d3c-8700-4c02-91bf-99016ba0f740

### How did axe testing go?
Went great within scope of the new page content; no issues found.  The issues found are in the global navigation.
- [X] SS confirmed that the global navigation issue is known. _(See comments in this ticket for details.)_

### Who completed the content zoom and reflow test?

Marci McGuire

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Testing went well.  Page reflowed as expected.

### Who completed the keyboard navigation test?

Marci McGuire

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

It went well.  Tab order made sense, focus indicator was visible, clicking Enter worked as expected.

### Do you have any other results to share?

- [X] WAVE Spot Check - no errors (only global issues previously indicated by Marci)
- [X] VoiceOver in Chrome - text on screen matches text announced; buttons and links provide a clear understanding; reading order and heading styles are correct.

**Comment by MarciMcGuire** [comment]
@briandeconinck - what is the best way to include Axe scan results in this issue?  Thank you!

**Comment by briandeconinck** [comment]
@MarciMcGuire Great question! I don't have a preferred format. Some teams have just included screenshots from the axe browser extension, some have copy/pasted text from the results panel of the browser extension. Whatever works best for you and gives me something to look at. And if you're not returning any issues from axe, you can just state that!

**Comment by sara-amanda** [comment]
@MarciMcGuire I took a look at your testing and also ran WAVE and VoiceOver.

**Comment by sara-amanda** [comment]
## RE: Global Nav Issue

### Historical Ticket: 34163
@laflannery shared this historical ticket @briandeconinck re: unique ARIA labels and `account-menu`
- Ticket [#34163](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34163) 
- **Current Status:** Open
- **Last Comment:** Jan. 27, 2022
- _[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)_
- **Issue:** HTML markup isn't valid. The value of an ARIA ID must be unique to prevent other instances from being overlooked by assistive technologies.
- **WCAG:** 1.3.1_A
- **[Experience Standards:](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards)** D-depends
- **Impact:** a11y

### Current Ticket: 54863
@MarciMcGuire 's scan and my re-check is finding is also relating to `account-menu` however ...
- Instead it is coming up as a _[Category Number 09, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)_
- **Issue**: [IDs used in ARIA and labels must be unique](https://dequeuniversity.com/rules/axe/4.6/duplicate-id-aria?application=AxeChrome)
- **Issue Detail:** Document has multiple elements referenced with ARIA with the same id attribute: `account-menu`
- **WCAG:** 4.1.1_A 
- **[Experience Standards:](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards)** LB - launch-blocking
- **Impact:** a11y 

## Potential Variable
### WCAG 2.2
**In the drafted WCAG 2.2 it says:** _This criterion 4.1.1 was originally adopted to address problems that Assistive Technology had directly parsing HTML. Assistive Technology no longer has any need to directly parse HTML and, consequently, these problems no longer exists. Accessibility errors failed by this criterion also fail other criteria. This criterion no longer has utility and is removed._

**_Would you like me to connect this documentation to the existing issue [#34163](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34163) and add any additional labels to it, since it is still under investigation?_** 

### Images for Reference
![Screenshot 2023-03-11 at 4.03.14 PM.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/99c41f73-63ce-4dc5-9996-5ce49a276d5a)
![Screenshot 2023-03-11 at 3.36.41 PM.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/4d71c89b-fa09-4449-a8d4-c494c4dd8f12)
